### PR TITLE
Added Butcher tableau of the rungekuttaSsc method to the gbode framework

### PR DESCRIPTION
**Pull request by bernhardbachmann**
see:
 Solving Stiff Systems of ODEs by Explicit Methods with Conformed Stability Domains
 Anton E. Novikov1     Mikhail V. Rybkov2     Yury V. Shornikov3     Lyudmila V. Knaub4
 EUROSIM 2016 & SIMS 2016

### CODE REVIEW

File: OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c [hunk 1]
```diff
@@ -288,6 +288,26 @@ void getButcherTableau_MS(BUTCHER_TABLEAU* tableau)
   tableau->isKRightAvailable = TRUE;
 }
 
+// TODO: Describe me
```
Comment (AnHeuermann) [inline OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c:291]: Could you add a super short description?
Maybe just link to Wiki: https://en.wikipedia.org/wiki/Heun%27s_method

File: OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c [hunk 2]
```diff
@@ -739,6 +759,27 @@ void getButcherTableau_IMPLEULER(BUTCHER_TABLEAU* tableau) {
...
```
Comment (AnHeuermann) [inline OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c:762]: Same, add a link to the method.

File: OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c [hunk 3]
```diff
@@ -1018,6 +1059,31 @@ void getButcherTableau_RK1214(BUTCHER_TABLEAU* tableau) {
   tableau->isKRightAvailable = FALSE;
 }
 
+// Solving Stiff Systems of ODEs by Explicit Methods with Conformed Stability Domains
+// Anton E. Novikov1     Mikhail V. Rybkov2     Yury V. Shornikov3     Lyudmila V. Knaub4
+// EUROSIM 2016 & SIMS 2016
```
Comment (AnHeuermann) [inline OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c:1064]: Make this a Doxygen doc comment

```suggestion
/** @brief Get Runge-Kutta Ssc Butcher tableau.
 * Solving Stiff Systems of ODEs by Explicit Methods with Conformed Stability Domains
 * From:  Anton E. Novikov1     Mikhail V. Rybkov2     Yury V. Shornikov3     Lyudmila V. Knaub4
 * EUROSIM 2016 & SIMS 2016
 *
 * @param tableau    Pointer to Butcher tableau to fill.
 */
```

**Review by AnHeuermann — CHANGES_REQUESTED** [review]
Add more documentation

File: OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c (hunk 1 above)
Comment (bernhardbachmann) [reply OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c]: I added a link to the general Wikipedia page on RK methods. There will come more details on the references later, also for the other methods.

File: OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c (hunk 2 above)
Comment (bernhardbachmann) [reply OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c]: dito

**Review by AnHeuermann — APPROVED** [review]

**Review by phannebohm — APPROVED** [review]

**Pull request merged**